docker-local/ml-testing-toolkit/spec_files/api_definitions/fspiop_1.1/mockRef.json

```diff
@@ -0,0 +1,83 @@
+[
```

Comment:
super nit: I like to delete all the base template stuff for ttk's if they aren't going to use these apis. don't feel the need to fix this.